commands/auth_e2e_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestAuthLoginWithTrailingSlashInInstanceURL() {
+	s.Run("URL with trailing slash", func() {
+		// cobra wont let us run a subcommand directly. When we try to `execute`
+		// the subcommand cobra executes the parent command
+		// Instead of calling RootCmd, with its various subcommands and options,
+		// we duplicate part of the the LoginCmd here
+		cmd := &cobra.Command{}
+		cmd.Flags().StringP("name", "n", "name", "Name for the credentials")
+		cmd.Flags().StringP("username", "u", SysadminUsername, "Username for the credentials")
+		cmd.Flags().StringP("password", "p", SysadminPass, "Password for the credentials")
+		cmd.Flags().StringP("access-token", "a", "", "Access token to use instead of username/password")
+		cmd.Flags().StringP("mfa-token", "m", "", "MFA token for the credentials")
+		cmd.Flags().Bool("no-activate", false, "If present, it won't activate the credentials after login")
+
+		// loginCmdf doesn't return an error in this case. It prints to stderr instead.
+		printer.Clean()
```

Comment:
nit: this clean could be moved to the beginning of the test case (right after the `s.Run`) for consistency.